pipeline/scraper/db.js

```diff
@@ -45,6 +45,34 @@ async function insertDev(dev) {
 }
 
 module.exports = {
```

Comment:
We're now using a DB class that has a constructor to initialise access to the Postgres DB based using accounts in the config. would recommend updating to bring it in line with the DB class export before merging.